src/main/java/Duke.java

```diff
@@ -1,10 +1,287 @@
+import exceptions.DeadlineParamsFormatException;
+import exceptions.EventParamsFormatException;
+import exceptions.TaskIndexNotFoundException;
+import task.Deadline;
+import task.Event;
+import task.Task;
+import task.Todo;
+
+import java.util.*;
+
 public class Duke {
-    public static void main(String[] args) {
+
+    static ArrayList<Task> tasksList = new ArrayList<Task>();
+
+    public static void sendLogo(){
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
     }
+
+    public static void sendGreeting(){
+        String greeting_word="------------------------------------------------------------\n"
```

Comment:
Try to not use string literals and try to define them with the `final` keyword to make it obvious that it is not going to change